### ISSUE TITLE
SMS_D.f09_g16.I1850CLM45CN.cori-knl_intel fails

### ISSUE THREAD

**Issue by ndkeen** [E3SM-Project/E3SM#3284]
This is the test (of many others) that is failing on GNU. I assumed it worked with intel, but looks like it does not.

```
SMS_D.f09_g16.I1850CLM45CN.cori-knl_intel fails
SMS_D_PMx1.f09_g16.I1850CLM45CN.cori-knl_intel passes
```

```
31:  ERROR: ERROR: ERROR in subgridAveMod.F90 at line 1039
31: Image              PC                Routine            Line        Source
31: e3sm.exe           00000000067C0466  Unknown               Unknown  Unknown
31: e3sm.exe           00000000048A91AB  shr_abort_mod_mp_         114  shr_abort_mod.F90
31: e3sm.exe           00000000048A9028  shr_abort_mod_mp_          61  shr_abort_mod.F90
31: e3sm.exe           00000000048AAD7D  shr_assert_mod_mp          95  shr_assert_mod.F90.in
31: e3sm.exe           00000000048AB110  shr_assert_mod_mp         112  shr_assert_mod.F90.in
31: e3sm.exe           0000000000DD6192  subgridavemod_mp_        1039  subgridAveMod.F90
31: e3sm.exe           0000000000BA1753  histfilemod_mp_hi        1362  histFileMod.F90
31: e3sm.exe           0000000000B8835E  histfilemod_mp_hi        1005  histFileMod.F90
31: e3sm.exe           00000000052C90E3  Unknown               Unknown  Unknown
31: e3sm.exe           00000000052809C0  Unknown               Unknown  Unknown
31: e3sm.exe           0000000005281FB4  Unknown               Unknown  Unknown
31: e3sm.exe           00000000052538D4  Unknown               Unknown  Unknown
31: e3sm.exe           0000000000B87B1B  histfilemod_mp_hi         998  histFileMod.F90
31: e3sm.exe           00000000008A9857  clm_driver_mp_clm        1413  clm_driver.F90
31: e3sm.exe           0000000000873D03  lnd_comp_mct_mp_l         509  lnd_comp_mct.F90
31: e3sm.exe           000000000045DCB7  component_mod_mp_         737  component_mod.F90
31: e3sm.exe           0000000000427529  cime_comp_mod_mp_        2607  cime_comp_mod.F90
31: e3sm.exe           0000000000445CB1  MAIN__                    133  cime_driver.F90
31: e3sm.exe           0000000000401ADE  Unknown               Unknown  Unknown
31: e3sm.exe           00000000068B6A4F  Unknown               Unknown  Unknown
31: e3sm.exe           00000000004019CA  Unknown               Unknown  Unknown
```

**Comment by amametjanov** [comment]
This is fixed by E3SM-Project/E3SM#3260, specifically commit https://github.com/E3SM-Project/E3SM/pull/3260/commits/164ab78ef8348e204c230009b5677a7fad137dbb. This does not happen anymore. Closing, please re-open otherwise.

**Comment by ndkeen** [comment]
Yes, this is now working.
